File: App_OBJC_SWIFT_INTEGRAFION/App_OBJC_SWIFT_INTEGRAFION/ViewController.m
//
//  ViewController.m
//  App_OBJC_SWIFT_INTEGRAFION
//
//  Created by cice on 5/9/16.
//  Copyright © 2016 Cice. All rights reserved.
//

#import "ViewController.h"

@interface ViewController ()

@end

@implementation ViewController

#pragma mark - LIFE VC
- (void)viewDidLoad {
    [super viewDidLoad];
    //Esta es la forma de hacer un print por consola
    NSLog(@"Estamos aqui chicos :)");
    // Do any additional setup after loading the view, typically from a nib.
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

#pragma mark - ACTIONS
- (IBAction)showNuevoSaludoOBJC:(id)sender {
    self.MySaludaMundoOBJCLBL.text = @"Hola a todos en OBJC";
}
@end

File: AppHolaMundo/AppHolaMundo/ViewController.swift
//
//  ViewController.swift
//  AppHolaMundo
//
//  Created by cice on 4/5/16.
//  Copyright © 2016 Cice. All rights reserved.
//

import UIKit

class ViewController: UIViewController {
    
    //MARK: - IBOUTLET
    
    @IBOutlet weak var mySaludaMundoLabel: UILabel!
    
    
    //MARK: - IBACTION
    
    
    @IBAction func myVengaSaludaACTION(sender: AnyObject) {
        mySaludaMundoLabel.text = "Hola Mundo!"
    }
    
    
    
    //MARK: - LIFE
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }


}

